config/application.rb

```diff
@@ -16,6 +16,7 @@ module BeastMode
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.1
+    config.active_record.schema_format = :sql
```

Comment:
Is there a reason that you switched to SQL schema? Something that cannot be done unless you do? Is it necessary if you want to create triggers, for example?

I'd prefer to not use SQL schemas, but if there's no option, I'll live with it.